### PR TITLE
WIP: Style who's on the list

### PR DESCRIPTION
**Pull request by waiyaki**
#### What does this PR do?
Adds WIP styling to who's on the list side panel.

#### Questions:

### CODE REVIEW

File: src/BrownBag/View.elm [hunk 1]
```diff
@@ -105,21 +105,19 @@ shuffleView =
...
 
-
--- TODO: Remove confusing function identifier
-
-
-presentersList : List BrownBagPresenter -> Html msg
-presentersList brownBags =
+whosOnTheList : List BrownBagPresenter -> Html msg
+whosOnTheList brownBags =
     brownBags
```
Comment (akabiru) [inline src/BrownBag/View.elm]: @waiyaki Could we change this variable to `undoneBrownbags` or something more expressive?

File: src/BrownBag/View.elm [hunk 2]
```diff
@@ -105,21 +105,19 @@ shuffleView =
...
         |> List.filter (\b -> b.status == NotDone)
         |> whosOnTheListView
 
 
 whosOnTheListView : List BrownBagPresenter -> Html msg
 whosOnTheListView brownBags =
```
Comment (akabiru) [inline src/BrownBag/View.elm]: Same here?

**Review by akabiru — CHANGES_REQUESTED** [review]
LGTM +1

File: src/BrownBag/View.elm (hunk 1 above)
Comment (waiyaki) [reply src/BrownBag/View.elm]: @akabiru Done.

**Review by akabiru — APPROVED** [review]
Awesome. Thanks bruh.👍🏾

**Pull request merged**